File: Programación Estructurada/Ejercicio3.js
import java.util.Scanner;

/**
 *
 * @author belen
 */
public class Ejercicio3 {

    /**
     * @param args the command line arguments
     */
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int edad;
        
        System.out.print("Ingresa tu edad: ");
        edad = Integer.parseInt(input.nextLine());
        
        if (edad < 12){
            System.out.print("Eres un Niño.");
        }
        if (edad >= 12 && edad <= 17){
            System.out.print("Eres un Adolescente.");
        }
        if (edad >= 18 && edad <= 59){
            System.out.print("Eres un Adulto.");
        }
        if (edad >= 60){
            System.out.print("Eres un Adulto mayor.");
        }
    }
    
}

File: Introduccion a POO /Ejercicio3.java
package ejercicio3;

/**
 *
 * @author belen
 */
public class Ejercicio3 {

    /**
     * @param args the command line arguments
     */
    public static void main(String[] args) {
        Libro lib = new Libro();
        
        
      lib.getTitulo();
      lib.getAutor();
        
      
      lib.setAnioPublicacion(7); // año invalido
      
      lib.getTitulo();
      lib.getAutor();
      
      lib.setAnioPublicacion(1943);
        
        
    }
    
}
